bin/retest.mjs

```diff
@@ -70,13 +70,7 @@ if (globsOrNames.some((item) => item.includes("*"))) {
     globsOrNames.map(
       (globOrName) =>
         new Promise((resolve, reject) => {
-          glob(globOrName, (err, files) => {
```

Comment:
Apparently this callback variant does not exist anymore in the version 10.3.10 (cf: https://github.com/isaacs/node-glob?tab=readme-ov-file#globpattern-string--string-options-globoptions--promisestring--path ). Only promise and sync options exist.